package.json

```diff
@@ -1,38 +1,33 @@
 {
-  "name": "top-rated-movie-list",
-  "version": "0.1.0",
-  "private": true,
+  "name": "react",
+  "version": "1.0.0",
+  "description": "React example starter project",
+  "keywords": [
+    "react",
+    "starter"
+  ],
+  "main": "src/index.js",
   "dependencies": {
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.5.0",
-    "@testing-library/user-event": "^7.2.1",
-    "bootstrap": "^4.5.0",
-    "react": "^16.13.1",
-    "react-bootstrap": "^1.0.1",
-    "react-dom": "^16.13.1",
-    "react-route": "^1.0.3",
-    "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.1"
+    "bootstrap": "4.5.0",
+    "react": "16.12.0",
+    "react-bootstrap": "1.0.1",
+    "react-dom": "16.12.0",
+    "react-router-dom": "5.2.0",
```

Comment:
recommend keep Semantic Versioning (with ^). You can check on this [Semver explained - why is there a caret (^) in my package.json?](https://bytearcher.com/articles/semver-explained-why-theres-a-caret-in-my-package-json/)